examples/argocd/main.tf

```diff
@@ -58,6 +55,15 @@ module "eks" {
   cluster_version                = "1.25"
   cluster_endpoint_public_access = true
 
+  # EKS Addons
+  cluster_addons = {
```

Comment:
doesn't Argo use PVs for Dex/Redis (in HA mode)? We might want to add the EBS CSI driver since thats required now starting in EKS 1.23+